lib/brainstem/api_docs.rb

```diff
@@ -0,0 +1,43 @@
+require 'active_support/configurable'
+
+module Brainstem
+  module ApiDocs
+    include ActiveSupport::Configurable
+
+    config_accessor(:base_path) do
+      File.expand_path("./brainstem_docs")
+    end
+
+    config_accessor(:controller_filename_pattern) do
+      File.join("controllers", "{{name}}_controller.{{extension}}")
```

Comment:
Also, if you have another preferred format, I'd be happy to use it. I debated both `printf` and ERB-style templating for those but this seemed like the most self-explanatory.